server/src/index.js

```diff
@@ -5,7 +5,7 @@ import Opencv from 'opencv';
 import { crop } from 'easyimage';
 
 import Image from './models/image';
-import saveImageFromCamera from '../lib/saveImage';
+import saveImage from '../lib/saveImage';
 // import gifOut from '../lib/gifOut';
```

Comment:
Shit, I need to clean this up, since we don't need it anymore. I'll add another task for myself for that.